resources/lang/en.json

```
@@ -0,0 +1,12 @@
+{
+        
+    "Posts" : "Posts",
+    "Add new post" : "Add new post",
+    "Action" : "Action",
+    "Edit" : "Edit",
+    "Delete" : "Delete",
+    "View all comments" : "View all comments",
+    "Title" : "Title",
+    "Author" : "Author"
```

```suggestion
    "view_all_comments" : "View all comments",
    "title" : "Title",
    "author" : "Author"
```
key nên viết ntn nhé

---

resolved